protocols/relay/src/priv_client/handler.rs

```diff
@@ -319,28 +319,12 @@ impl ConnectionHandler for Handler {
         }
     }
 
-    fn connection_keep_alive(&self) -> KeepAlive {
+    fn connection_keep_alive(&self) -> bool {
         if self.reservation.is_some() {
-            return KeepAlive::Yes;
+            return true;
```

Comment:
Couldn't you just return `self.reservation.is_some()` directly?

Reply:
fixed 3d10eb4